src/main/java/org/fenixedu/academic/domain/Person.java

```diff
@@ -1101,6 +1101,15 @@ public static Person readPersonByEmailAddress(final String email) {
         return emailAddress != null && emailAddress.getParty().isPerson() ? (Person) emailAddress.getParty() : null;
     }
 
+    public static Stream<Person> readPeopleByEmailAddress(final String email) {
```

Comment:
I think the `Person` class is bloated enough as it is, I see no need to add a method that is basically a wrapper, and that is only invoked in a single place.